WordPress/gutenberg
# Inline commenting - Focus the reply field when clicking on the comment toolbar button

Part of - https://github.com/WordPress/gutenberg/issues/66377

## What? & Why?

The comment sidebar opens and the comment board becomes focused upon clicking the comment toolbar button when using the inline commenting feature. However, when the comment sidebar is already open, it does nothing.

## How?

This PR implement action to open up the reply field on comment toolbar button.

## Testing Instructions

- Enable experimental inline commenting option under `Gutenberg` -> `Experiments`
- Edit any post/page
- Add any block, Click on more menu in block toolbar, Look for comment option.
- It would open the thread in sidebar, Where you can add comment.
- Reset focus on block, click comment toolbar button
- Notice reply field is visible

## Screenshots or screencast
